code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -139,7 +139,7 @@
 
 /obj/item/reagent_containers/food/snacks/carrotfries
 	name = "carrot fries"
-	desc = "Tasty fries from fresh Carrots."
+	desc = "Tasty fries from fresh carrots."
```

Comment:
clearly this is actually referring to your pig not an actual carrot, so the fix would be to capitalize Carrot fries